app/services/foreman_templates/template_importer.rb

```diff
@@ -25,6 +25,21 @@ def initialize(args = {})
     end
 
     def import!
+      if @repo.start_with?('http://', 'https://', 'git://')
+        import_from_git
+      elsif repo.start_with?('/')
```

Comment:
I'm not saying *default* to relative - absolute should definitely be the recommended use. However, I can think of a few situations where the absolute path may not be known - automation in general (Jenkins, tests, other CI pipelines) tend to use temporary directories. Also, if you're doing any kind of binding (chroots, or cross-mounted filesystems) then again it can be tricky to figure out the absolute path.

I'm open to leaving it absolute-only for now, but I can see people wanting it later. Does it hurt to support relative paths, along with a "here be dragons" warning?